doc/man7/OSSL_PROVIDER-legacy.pod

```diff
@@ -0,0 +1,92 @@
+=pod
+
+=head1 NAME
+
+OSSL_PROVIDER-legacy - OpenSSL legacy provider
+
+=head1 DESCRIPTION
+
+The OpenSSL legacy provider supplies OpenSSL implementations of algorithms
+that have been deemed legacy.  Such algorithms have commonly fallen out of
+use, have been deemed insecure by the cryptography community, or something
+along those lines.
```

Comment:
"or something along those lines" is very informal. Perhaps "or similar"?

Reply:
Changed